cmake/helpers.cmake

```diff
@@ -91,18 +91,24 @@ function(add_umf_target_compile_options name)
         target_compile_options(
             ${name}
             PRIVATE -fPIC
+                    # -fPIE
                     -Wall
+                    -Wextra
+                    -Werror
                     -Wpedantic
                     -Wempty-body
                     -Wunused-parameter
+                    -Wformat
+                    -Wformat-security
+                    -fstack-protector-strong
+                    -D_FORTIFY_SOURCE=2
+                    # -flto -fvisibility=hidden
                     $<$<CXX_COMPILER_ID:GNU>:-fdiagnostics-color=auto>)
         if(CMAKE_BUILD_TYPE STREQUAL "Release")
             target_compile_definitions(${name} PRIVATE -D_FORTIFY_SOURCE=2)
         endif()
         if(UMF_DEVELOPER_MODE)
-            target_compile_options(
-                ${name} PRIVATE -Werror -fno-omit-frame-pointer
-                                -fstack-protector-strong)
+            target_compile_options(${name} PRIVATE -fno-omit-frame-pointer)
```

Comment:
Change the description of the UMF_DEVELOPER_MODE option as it doesn't enable Werror anymore.

Reply:
done